client/src/pages/ProfileSetting/Profile/Profile.tsx

```diff
@@ -23,40 +24,76 @@ const TabPanel = (props: TabPanelProps) => {
       {...other}
     >
       {value === index && (
-        <Box p={3}>
-          <Typography>{children}</Typography>
-        </Box>
+        <Container>
+          <Box p={3}>{children}</Box>
+        </Container>
       )}
     </div>
   );
 };
 
-const mockContests = [
-  {
-    imgSrc: mockContestPic,
-    imgCount: 25,
-    headline: 'Lion tattoo concept in minimal style',
-    byline: 'Looking for a cool simple ideas',
-    prizeAmt: 150,
-  },
-  {
-    imgSrc: mockContestPic,
-    imgCount: 9,
-    headline: 'Anchors, like all the pirates',
-    byline: 'Artistic anchors or anchor related art',
-    prizeAmt: 75,
-  },
-];
-
 const Profile: FC = (): JSX.Element => {
   const classes = useStyles();
-  const { loggedInUser } = useAuth();
   const [value, setValue] = useState(0);
+  const [contests, setContests] = useState<ContestAPIData[]>([]);
+
+  useEffect(() => {
```

Comment:
I know we discussed about this in the morning for a bit, having a look at the code I do notice that each time the Profile components mounts, all the contests for the user are being fetched and the sorting is being done below. Maybe consider using a context(existing/new) instead?

Reply:
I'm not sure what you mean.  Let's discuss briefly this morning in the stand up.

Reply:
I made a context for the contests, but still have to loop over the contests to get a count and a sample image.